go.mod

```diff
@@ -143,3 +143,9 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace (
+	// Temporarily divert repo for updated/extended NRI.
+	github.com/containerd/nri => github.com/klihub/nri v0.0.0-20220912062812-c1cf5a4d5b84
```

Comment:
If this will still be there when we merge, can you add a comment to the actual `require` above to mention that it's replaced?

Reply:
Added a comment about the replace to the original require.